pkg/datapath/loader/base.go

```diff
@@ -109,7 +109,6 @@ func writePreFilterHeader(preFilter *prefilter.PreFilter, dir string) error {
 	fw := bufio.NewWriter(f)
 	fmt.Fprint(fw, "/*\n")
 	fmt.Fprintf(fw, " * XDP devices: %s\n", strings.Join(option.Config.GetDevices(), " "))
-	fmt.Fprintf(fw, " * XDP mode: %s\n", option.Config.ModePreFilter)
```

Comment:
We could report `option.Config.NodePortAcceleration` here instead. Not sure whether it is really useful, though.

Reply:
> We could report `option.Config.NodePortAcceleration` here instead. Not sure whether it is really useful, though.

Ack, let's keep this alive for debugging.